### PR TITLE
[fix bug 1166272] Check if an email was already sent to a user.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

File: remo/events/tasks.py [hunk 1]
```diff
@@ -22,11 +24,18 @@ def notify_event_owners_to_input_metrics():
...
-                   .format(event.name))
-        template = 'email/event_creator_notification_to_input_metrics.txt'
-        data = {'event': event}
-        send_remo_mail(subject=subject, email_template=template,
-                       recipients_list=[event.owner.id], data=data)
-        ActionItem.create(instance=event)
+        # Before sending an email check that an action item already exists.
+        # If it does, then we have already sent this email.
+
+        action_item = ActionItem.objects.filter(content_type=event_model,
```
Comment (johngian) [inline remo/events/tasks.py]: Instead of iterating over events you could just filter events without existing action items.

**Comment by johngian** [comment]
r+

**Comment by johngian** [comment]
r+wc

**Comment by johngian** [comment]
lgtm r+

**Pull request merged**